doc/classes/EditorImportPlugin.xml

```diff
@@ -163,18 +163,18 @@
 				Gets whether the import option specified by [param option_name] should be visible in the Import dock. The default implementation always returns [code]true[/code], making all options visible. This is mainly useful for hiding options that depend on others if one of them is disabled.
 				[codeblocks]
 				[gdscript]
-				func _get_option_visibility(option, options):
+				func _get_option_visibility(path, option, options):
```

Comment:
```suggestion
				func _get_option_visibility(path, option_name, options):
```

Reply:
Thanks, updated in the signature and function body to option_name

Reply:
You may have forgotten to push the change

Reply:
Sorry